### PR TITLE
14.0 mig pos backend communication rebased

### PR DESCRIPTION
**Pull request by sebastienbeau**

### CODE REVIEW

**Review by hparfr — APPROVED** [review]
LGTM

File: pos_backend_communication/static/src/js/back.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+odoo.define("pos_backend_communication.back", function (require) {
+    "use strict";
+    var tools = require("pos_backend_communication.tools");
```
Comment (ivantodorovich) [inline pos_backend_communication/static/src/js/back.js]:
```suggestion
    const tools = require("pos_backend_communication.tools");
```

Non blocking, but I'd use `const` for all the `var`s

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Can you squash all your Migration commits into a single "[MIG] pos_backend_communication: Migration to 14.0" commit?

**Comment by ivantodorovich** [comment]
It'd also be good to take the opportunity to squash some useless commits to keep a clean history:

image

**Comment by sebastienbeau** [comment]
@ivantodorovich done

**Review by ivantodorovich — APPROVED** [review]
Thanks! ☺️

**Comment by sebastienbeau** [comment]
thanks

**Pull request merged**